hydra/conf/hydra/hydra_logging/notset.yaml

```diff
@@ -0,0 +1,5 @@
+# @package _group_
```

Comment:
I'm not sure if `notset` is a good name since `NOTSET` is actually a logging level https://docs.python.org/3.8/library/logging.html

maybe something like `root_null`?

Reply:
How about "none"?
```
hydra/hydra_logging/none.yaml
hydra/job_logging/none.yaml
```

Reply:
`disable.yaml`

`hydra/hydra_logging=disable`

Reply:
`disabled.yaml` already exists :) https://github.com/facebookresearch/hydra/blob/master/hydra/conf/hydra/hydra_logging/disabled.yaml

Reply:
ohh thanks for catching this :)